ding2.install

```diff
@@ -503,3 +503,14 @@ function ding2_update_7040() {
   ding2_translation_update();
 }
 
+/**
+ * Delete {system} records for ting_carousel modules.
+ *
+ * @see https://www.drupal.org/node/2487215
+ */
+function ding2_update_7041() {
+    db_delete('system')
+    ->condition('name', 'ting_carousel')
+    ->condition('type', 'module')
+    ->execute();
+}
```

Comment:
This seems rather crude. Why not use the normal disable + uninstall procedures?

Reply:
The module have been remove in this PR. So it's not possible to call the module disable. The old module did not have any hook_uninstall(). So this was the solution.

Reply:
:+1: - perhaps add a comment about this.

I would also consider implementing a helper function á la `ding2_remove_module($modules)` which wrapped the query above and was documented appropriately.

Reply:
done